system/threads.cpp

```diff
@@ -298,6 +344,20 @@ void spawnUtility()
 	utilityWorker->addEventListener("message", callStart);
 }
 
+// This function is called from pthread_exit in musl.
+[[cheerp::genericjs]]
```

Comment:
add a `[[noreturn]]` attribute to functions that never return, like this and `closeMainThreadAsWorker`

Reply:
Done